diff --git a/data/com.github.Matoking.protontricks.metainfo.xml b/data/com.github.Matoking.protontricks.metainfo.xml
index ff53f1d..a9df4de 100644
--- a/data/com.github.Matoking.protontricks.metainfo.xml
+++ b/data/com.github.Matoking.protontricks.metainfo.xml
@@ -3,7 +3,6 @@
   <id>com.github.Matoking.protontricks</id>
   <name>Protontricks</name>
   <summary>Apps and fixes for Proton games</summary>
-  <icon type="stock">wine</icon>
   <provides>
     <binary>protontricks</binary>
     <binary>protontricks-launch</binary>
diff --git a/src/protontricks/data/share/applications/protontricks-launch.desktop b/src/protontricks/data/share/applications/protontricks-launch.desktop
index 56112d7..66cac55 100644
--- a/src/protontricks/data/share/applications/protontricks-launch.desktop
+++ b/src/protontricks/data/share/applications/protontricks-launch.desktop
@@ -5,5 +5,5 @@ Type=Application
 Terminal=false
 NoDisplay=true
 Categories=Utility
-Icon=wine
+Icon=com.github.Matoking.protontricks
 MimeType=application/x-ms-dos-executable;application/x-msi;application/x-ms-shortcut;
diff --git a/src/protontricks/data/share/applications/protontricks.desktop b/src/protontricks/data/share/applications/protontricks.desktop
index ec0e964..bfd53c7 100644
--- a/src/protontricks/data/share/applications/protontricks.desktop
+++ b/src/protontricks/data/share/applications/protontricks.desktop
@@ -5,5 +5,5 @@ Comment=A simple wrapper that does winetricks things for Proton enabled games
 Type=Application
 Terminal=false
 Categories=Utility;
-Icon=wine
+Icon=com.github.Matoking.protontricks
 Keywords=Steam;Proton;Wine;Winetricks;
diff --git a/src/protontricks/gui.py b/src/protontricks/gui.py
index 1c02496..767df36 100644
--- a/src/protontricks/gui.py
+++ b/src/protontricks/gui.py
@@ -294,7 +294,7 @@ def select_steam_app_with_gui(steam_apps, steam_path, title=None):
     def _get_yad_args():
         return [
             "yad", "--list", "--no-headers", "--center",
-            "--window-icon", "wine",
+            "--window-icon", "/app/share/icons/hicolor/scalable/apps/com.github.Matoking.protontricks.svg",
             # Disabling markup means we won't have to escape special characters
             "--no-markup",
             "--search-column", "2",
@@ -461,7 +461,7 @@ def prompt_filesystem_access(paths, show_dialog=False):
         ignore = show_text_dialog(
             title="Protontricks",
             text=message,
-            window_icon="wine",
+            window_icon="/app/share/icons/hicolor/scalable/apps/com.github.Matoking.protontricks.svg",
             cancel_label="Close",
             ok_label="Ignore, don't ask again",
             add_cancel_button=True
